go-controller/pkg/util/pod_annotation.go

```diff
@@ -51,6 +51,9 @@ const (
 	OvnPodAnnotationName = "k8s.ovn.org/pod-networks"
 	// DefNetworkAnnotation is the pod annotation for the cluster-wide default network
 	DefNetworkAnnotation = "v1.multus-cni.io/default-network"
+	// OvnUDNIPAMClaimName is used for workload owners to instruct OVN-K which
+	// IPAMClaim will hold the allocation for the workload
+	OvnUDNIPAMClaimName = "k8s.ovn.org/ovn-udn-ipamclaim-reference"
```

Comment:
@trozet / @tssurya 

let's discuss the annotation value.

Reply:
This could just be `k8s.ovn.org/ipamclaim-reference`

Reply:
really ? I think it should somehow indicate this is meant to tell ovn-k of what is the claim to be used for the primary UDN. 

I agree dropping the `ovn` should happen ... awfully redundant.

Reply:
Done.

Reply:
@qinqon we'll need to update the client for the e2e tests to pass ...